packages/injected/src/icons/mew-wallet.ts

```diff
@@ -0,0 +1,24 @@
+export default `
```

Comment:
This svg isnt needed and can be removed as MEW is not an injected-wallet package option